client/me/purchases/remove-purchase/index.jsx

```diff
@@ -117,32 +228,100 @@ const RemovePurchase = React.createClass( {
 				className="remove-purchase__dialog"
 				isVisible={ this.state.isDialogVisible }
 				onClose={ this.closeDialog }>
-				<h1>{ this.translate( 'Remove %(productName)s', { args: { productName } } ) }</h1>
-				{ this.renderDialogText() }
+				<FormSectionHeading>{ this.translate( 'Remove %(productName)s', { args: { productName } } ) }</FormSectionHeading>
+				{ this.renderDomainDialogText() }
 			</Dialog>
 		);
 	},
 
-	renderDialogText() {
+	renderDomainDialogText() {
 		const purchase = getPurchase( this.props ),
 			productName = getName( purchase );
 
-		if ( isDomainRegistration( purchase ) ) {
-			return (
-				<p>
-					{
-						this.translate( 'This will remove %(domain)s from your account.', {
-							args: { domain: productName }
-						} )
-					}
-					{ ' ' }
-					{ this.translate( 'By removing, you are canceling the domain registration. This may stop you from using it again, even with another service.' ) }
-				</p>
-			);
+		return (
+			<p>
+				{
+					this.translate( 'This will remove %(domain)s from your account.', {
```

Comment:
It looks like this wasn't your code originally, but why not translate this whole block of text together? like:
```js
this.translate( 'This will remove %(domain)s from your account. By removing, ' +
'you are canceling the domain registration. This may stop you from using it ' +
'again, even with another service.', {
  args: { domain: productName }
}
```
I don't know if it's worth changing now, since it may already have been translated as separate strings (could check glotpress to see if it's been translated). But in general, I think we would want this to be translated as one block to give translators more flexibility in how they translate the whole thing.